diff --git a/proton-c/bindings/perl/CMakeLists.txt b/proton-c/bindings/perl/CMakeLists.txt
index 1356ccea..ed61d864 100644
--- a/proton-c/bindings/perl/CMakeLists.txt
+++ b/proton-c/bindings/perl/CMakeLists.txt
@@ -56,18 +56,9 @@ list(APPEND SWIG_MODULE_cproton_perl_EXTRA_DEPS
 swig_add_module(cproton_perl perl perl.i)
 swig_link_libraries(cproton_perl ${BINDING_DEPS} ${PERL_LIBRARY})
 
-if ((${CMAKE_MAJOR_VERSION} EQUAL 2) AND (${CMAKE_MINOR_VERSION} LESS 8))
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cproton_perl.so
+install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cproton_perl.so
         DESTINATION ${PERL_VENDORARCH_DIR}/auto/cproton_perl
-        COMPONENT Perl
-        )
-else()
-  install(FILES ${CMAKE_CURRENT_BINARY_DIR}/libcproton_perl.so
-        RENAME cproton_perl.so
-        DESTINATION ${PERL_VENDORARCH_DIR}/auto/cproton_perl
-        COMPONENT Perl
-        )
-endif ((${CMAKE_MAJOR_VERSION} EQUAL 2) AND (${CMAKE_MINOR_VERSION} LESS 8))
+        COMPONENT Perl)
 
 install(FILES ${CMAKE_CURRENT_BINARY_DIR}/cproton_perl.pm
         DESTINATION ${PERL_VENDORARCH_DIR}
